modules/service-account/outputs.tf

```diff
@@ -1,3 +1,27 @@
+output "primary_cluster" {
+  value = google_service_account.primary_cluster
+
+  description = "The service account to be associated with the primary cluster."
+}
+
+output "internal_load_balancer" {
+  value = google_service_account.internal_load_balancer
+
+  description = "The service account to be associated with the internal load balancer."
+}
+
+output "secondary_cluster" {
```

Comment:
same as the comment for https://github.com/hashicorp/terraform-google-terraform-enterprise/pull/54/files#diff-4824eea5aa6965c3b168b63f43a77980R1

Reply:
Fixed by https://github.com/hashicorp/terraform-google-terraform-enterprise/pull/54/commits/16f7c3fba5e9d24ed16d66ffd9d78183072bf957.